HesoyamHospital/IntegrationAdapter/PrescribedMedicineReport/specifications/andol.txt

```diff
@@ -0,0 +1 @@
+neki sastav
```

Comment:
What happens when we have over 10000 medicines in our system? Does it mean we will have a new txt file for every medicine? Is there any reason why you didn't use a database but chose to use text files?